src/common/registry/storage/operator.storage.ts

```diff
@@ -24,7 +24,7 @@ export class RegistryOperatorStorageService {
       .from<RegistryOperator>('registry_operator')
       .where(where)
       .orderBy([
-        { column: 'operatorIndex', order: 'asc' },
+        { column: 'moduleAddress', order: 'asc' },
```

Comment:
I find we dont need order by moduleAddress , because we never select operators for all modules in the same request

Reply:
removed